docs/kubernetes/overlays.md

```diff
@@ -0,0 +1,125 @@
+# Overview of per kubernetes minor version overlays
```

Comment:
Just a thought - I'm curious if this would make more sense living in the "development" page or some other "release" page as this info seems to be most relevant when one is trying to release something new (eg. new feature, new image, etc.). The word "overlays" isn't something that I would intuitively search for when hitting those cases without knowing that overlays existed in this component.

Reply:
Agree. I have moved it to docs/release.